core/src/test/java/io/questdb/test/griffin/TimeFrameRecordCursorTest.java

```diff
@@ -326,11 +326,6 @@ class TestCase {
                         "create table filtered_t (i int, ts timestamp) timestamp(ts) partition by day",
                         "filtered_t where i > 0"
                 ),
-                new TestCase(
```

Comment:
nit: might be a good idea to keep the test case, but rewrite the query in a non-intrinsics friendly way, e.g. `time_filtered_t where ts in '1970-01-13' or ts = '1970-05-14T16:00:02.000000Z'`